content/20.treatments.md

```diff
@@ -136,19 +136,46 @@ Summarize the state of the antiviral approach.
 
 #### Protease Inhibitors
 
-Why it may be useful
+The inhibition of virus proteases could be a promising approach to stop to the replication of the pathgen.
 
-##### Protease Inhibitor 1
+##### N3
+N3 is an inhibitor of Mpro, a 33.8-kDa SARS-CoV-2 protease that is involved in viral replication and transcriptipn.
 
 ###### Anticipated Mechanism
+N3 inhibits Mpro through binding to its substrate pocket.
 
 ###### Current Evidence
 
-A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+N3 was first designed computationally to bind in the substrate binding pocket of SARS-CoV and MERS-CoV Mpro proteases. 
+Consequently, the structure N3-bound SARS-CoV-2 Mpro has been solved [@doi:10.1038/s41586-020-2223-y], which confirmed the computational prediction.
+Finally, N3 reduced the viral in samples taken from patients.
 
 ###### Summary
 
-Summarize the state of the antiviral approach.
+N3 is a computationally designed molecule that inhibits the viral transcription through inhibiting Mpro.
+Although N3 is a strong inhibitor of SARS-CoV-2 in vitro, its safety and efficacy have to be tested in healthy volunteers and patients.
+
+##### Ebselen
+Ebselen identified as Mpro protease inhibitor.
+It is currently investigated as an anti-oxidant drug [@doi:10.1007/s11033-014-3417-x].
+
+###### Anticipated Mechanism
+Ebselen inhibits Mpro through binding to its substrate pocket.
+
+###### Current Evidence
+
+After the design and confirmation of N3 as a highly potent Michael acceptor inhibitor and the identification of Mpro structure [@doi:10.1038/s41586-020-2223-y.@doi:10.1126/science.abb3405], 10000 compounds were screened for their in vitro anti-Mpro activity.
+The six leads that were identified were Ebselen, Disulfiram, Tideglusib, Carmofour, PX-12.
+However when the compounds were further assayed on patient viral samples, Ebselen has the strongest potency in reducing the viral load.
```

Comment:
I like the new text below summarizing Ebselen.  In this sentence, perhaps it could be rephrased to something like
> However, the authors noted concerns about the potential promiscuity of these compounds.
> When the compounds were further assayed on patient viral samples, Ebselen has the strongest potency in reducing the viral load.

I'm not making this as a GitHub suggestion because we should go back to the original paper to see what specifically the authors said about these compound as promiscuous binders.

Reply:
Thanks for looking into this @agitter 
The authors state that ' In general, such molecules are expected to be promiscuous
binders and therefore, as they stand, may have limited potential as
drug leads.' 
Would like to make your addition as a github change?

Reply:
I just saw your commit, the wording is much better.
`However, the authors cautioned that these compounds are likely promiscuous binders, which would diminish their therapeutic potential.`